R/repr_htmlwidget.r

```diff
@@ -11,14 +110,14 @@ repr_text.htmlwidget <- function(obj, ...) 'HTML widgets cannot be represented i
 
 #' @name repr_*.htmlwidget
 #' @export
-repr_html.htmlwidget <- function(obj, ...) {
-	if (!requireNamespace('htmlwidgets', quietly = TRUE))
-		stop('repr_html.htmlwidget called without loadable htmlwidgets')
-	
-	htmlfile <- tempfile(fileext = '.html')
-	on.exit(unlink(htmlfile))
-	
-	htmlwidgets::saveWidget(obj, htmlfile)
-	
-	readChar(htmlfile, file.info(htmlfile)$size)
-}
+repr_html.htmlwidget <- embed_tags
+
+#' #' @name repr_*.shiny.tag
```

Comment:
This looks wrong. If you want to comment out things, please use `#`, as `#'` means “roxygen docs or directives”